week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -48,17 +65,52 @@ function exerciseTwo(shopping) {
             alreadyRead: true
         }
     ];
-
     Iterate through the array of books.
     - For each book, create a <p> element with the book title and author and append it to the page.
     - Use a <ul> and <li> to display the books.
     - Add an <img> to each book that links to a URL of the book cover.
     - Change the style of the book depending on whether you have read it (green) or not (red).
-
     The end result should look something like this: https://hyf-js2-week1-makeme-ex1-demo.herokuapp.com/
 **/
 function exerciseThree(books) {
   //Write your code in here
+  let content = document.querySelector("#content");
+  let createUl = document.createElement("ul");
+  content.appendChild(createUl);
+
+  createUl.style.display = "flex";
+  createUl.style.justifyContent = "space-around";
+  createUl.style.listStyleType = "none";
+
+  books.forEach((x) => {
+    let createLi = document.createElement("li");
+    createUl.appendChild(createLi);
+    let createP = document.createElement("p");
+    createLi.appendChild(createP);
+    createP.textContent = `${x.title} - ${x.author} `;
+
+    let createImage = document.createElement("img");
+    createLi.appendChild(createImage);
+    let createSrc = document.createAttribute("src");
```

Comment:
This works, but we can actually do something a little simpler, with:
```javascript
createImage.src = "https://image.ebooks.com/previews/095/095767/095767915/095767915-hq-168-80.jpg";
```

then we don't need to call `createAttribute` or `setAttributeNode` or the `.value` :)

Similarly for height (though ideally we'd set that as a CSS property, rather than an HTML attribute, like you do with `backgroundColor`)